src/Tools/dotnet-monitor/Auth/ApiAuthenticationOptions.cs

```diff
@@ -10,7 +13,14 @@ namespace Microsoft.Diagnostics.Tools.Monitor
 {
     internal sealed class ApiAuthenticationOptions
     {
+        [Description("API Key in Hashed form. Each byte should be two hex based digits.")]
+        [RegularExpression("[0-9a-fA-F]+")]
+        [MinLength(64)]
+        [Required]
         public string ApiKeyHash { get; set; }
+
+        [Description("Type of Hash Algorithm used to computed ApiKeyHash. Typically SHA256.  SHA1 and MD5 are not allowed.")]
```

Comment:
nit: "hash algorithm" should be lower case. "compute" without the "d" on the end. Remove the double space between the seconds and third sentences. Maybe put single quotes around SHA256, SHA1, and MD5. Some other minor changes:

"The hash algorithm used to compute ApiKeyHash, typically 'SHA256'. 'SHA1' and 'MD5' are not allowed."